src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -0,0 +1,126 @@
+import { loc, View, createCallout } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
+
+const SHOW_RESEND_TIMEOUT = 30000;
```

Comment:
this has been used at 3 places now. let me move into share file. e.g. `../../utils/Constants.js`